homeassistant/components/alexa/handlers.py

```diff
@@ -924,3 +927,76 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return response
+
+
+@HANDLERS.register(("Alexa.ChannelController", "SkipChannels"))
+async def async_api_skip_channels(hass, config, directive, context):
+    """Process a skip channel request."""
+    # media_player channel up/down service does not support specifying steps
+    # each component handles it differently e.g. via config.
+    # For now we use the channelCount returned to figure out if we
+    # should step up/down
+    channel_count = directive.payload["channelCount"]
+    entity = directive.entity
+
+    data = {ATTR_ENTITY_ID: entity.entity_id}
+
+    if channel_count > 0:
+        await hass.services.async_call(
+            entity.domain,
+            SERVICE_MEDIA_NEXT_CHANNEL,
+            data,
+            blocking=False,
+            context=context,
+        )
+    elif channel_count < 0:
+        await hass.services.async_call(
+            entity.domain,
+            SERVICE_MEDIA_PREVIOUS_CHANNEL,
+            data,
+            blocking=False,
+            context=context,
+        )
+
+    return directive.response()
+
+
+@HANDLERS.register(("Alexa.ChannelController", "ChangeChannel"))
+async def async_api_change_channel(hass, config, directive, context):
+    """Process a change channel request."""
+    # media_player channel service service does not support
+    # specifying channel name but only number
+
+    channel = directive.payload["channel"]
+    channel_number = 0
+    channel_name = ""
+
+    if "number" in channel:
+        channel_number = int(channel["number"])
+    elif "callSign" in channel:
+        channel_name = channel["callSign"]
+    elif "alternateCallSign" in channel:
+        channel_name = channel["alternateCallSign"]
+
+    if (
+        channel_number == 0
+        and not channel_name
+        and "channelMetadata" in directive.payload
+    ):
+        channel_metadata = directive.payload["channelMetadata"]
+        if "name" in channel_metadata:
+            channel_name = channel_metadata["name"]
+
+    entity = directive.entity
+
+    data = {
+        ATTR_ENTITY_ID: entity.entity_id,
+        media_player.const.ATTR_MEDIA_CHANNEL: int(channel_number),
+        media_player.const.ATTR_MEDIA_CHANNEL_NAME: channel_name,
+    }
+
+    await hass.services.async_call(
+        entity.domain, SERVICE_MEDIA_CHANNEL_SET, data, blocking=False, context=context
+    )
+
+    return directive.response()
```

Comment:
```suggestion
    response = directive.response()
    response.add_context_property(
        {
            "namespace": "Alexa.ChannelController",
            "name": "channel",
            "value": {
              "number": "1234",
              "callSign": "callsign1",
              "affiliateCallSign": "callsign2"
            }
        )
        
    return response
```
According the the Alexa API:

Same as the SkipChannels directive.